app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<header class="bg-sky-500 py-4 text-center w-96">
+  <h2 class="text-xl text-white uppercase">Categories</h2>
+</header>
+
+<div class="mt-5">
+  <%= render 'shared/notice' %>
+</div>
+
+<div class="mx-5 items-center mb-10 pt-20">
+  <% @categories.each_with_index do |category, index| %>
+    <%= link_to category_expenses_path(category.id), class: do %>
+    <div class="flex justify-between bg-white rounded-md shadow-lg p-4 mb-6">
+      <div>
+        <%= image_tag category.icon, alt: 'icon', class: "w-24" %>
+      </div>
```

Comment:
- [ ] When I try to sign up and I click on the `sign up button, it gives me this error below. I recommend fixing the syntax error so the categories index page opens without errors:

<img width="773" alt="image" src="https://user-images.githubusercontent.com/77190203/164892610-dabad747-a91e-4d88-820a-ab4b1dc9e8cf.png">

More details:

<img width="746" alt="image" src="https://user-images.githubusercontent.com/77190203/164892953-5de4412f-da6b-4d43-9754-bd47c8729c52.png">

Reply:
The Signup Button works well on my end here. I just signed up and created this category in the screenshot below
![image](https://user-images.githubusercontent.com/50941074/164896320-a596bc7d-c1df-4e8c-a72a-1194c9d713ba.png)